src/cephadm/tests/test_cephadm.py

```diff
@@ -2323,4 +2323,40 @@ def test_ip_in_subnet(self):
 
         # invalid IPv6 and valid subnets list
         with pytest.raises(Exception):
-            rc = cd.ip_in_sublets('fe80:2030:31:24', 'fe80::/64')
\ No newline at end of file
+            rc = cd.ip_in_sublets('fe80:2030:31:24', 'fe80::/64')
+
+
+class TestSysctl:
+    @mock.patch('cephadm.sysctl_get')
+    def test_filter_sysctl_settings(self, sysctl_get):
+        ctx = cd.CephadmContext()
+        input = [
+            # comment-only lines should be ignored
+            "# just a comment",
+            # As should whitespace-only lines",
+            "   \t ",
+            "   =  \t  ",
+            # inline comments are stripped when querying
+            "something = value # inline comment",
+            "fs.aio-max-nr = 1048576",
+            "kernel.pid_max = 4194304",
+            "vm.lowmem_reserve_ratio = 256\t256\t32\t0\t0",
+            "  vm.max_map_count       =            65530    ",
```

Comment:
Just a simple suggestion, plz ignore if you think the coverage is already good :)

plz consider including this line `vm.max_map_count       =            xxxxx ` in the desired output (by changing its value), this way we make sure the code is really filtering and not only returning always the first value. So the output should be:

```
assert result == [
            "fs.aio-max-nr = 1048576",
             "vm.max_map_count = xxxxx"
        ]

```

Reply:
Added one more just for you 😄